### PR TITLE
Restyle Cancel buttons to be 'link' variant and make buttons left alligned in modal footers

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
Some examples of the restyles buttons:
Screen Shot 2020-03-24 at 12 06 51
Screen Shot 2020-03-24 at 12 01 40
Screen Shot 2020-03-24 at 11 25 59
Screen Shot 2020-03-24 at 11 09 01

**Comment by marusak** [comment]
Nice! :)
I found one more case:
onecase
And not sure if it should also look like link when there is only one button?
cancelonce
And tests are unhappy :)

**Comment by garrett** [comment]
> Some examples of the restyled buttons:
> Screen Shot 2020-03-24 at 12 06 51

_Aside_: Oh, that merged button thing is a bit of an oversight visually, I think. Hmm.. Minor nitpick. (_Not_ related to this PR. That's a PF3 override thing.)

**Comment by garrett** [comment]
Example dialog has incorrect button order. Not that it really matters, but... :rofl: 

Screenshot_2020-03-25 Rain

**Comment by garrett** [comment]
Docker is a bit strange:
Screenshot_2020-03-25 Docker Containers - Rain
Screenshot_2020-03-25 Docker Containers - Rain(1)
Screenshot_2020-03-25 Docker Containers - Rain(2)

I know we're deprecating it, but this kind of breaks the UI a bit for existing stable distro releases.

I guess the tag should go on a line _above_ the buttons now? It won't make sense on the right of the buttons... and it's odd where it is now.

**Comment by KKoukiou** [comment]
@garret the new docker image download dialog:

Screen Shot 2020-03-25 at 13 11 50

**Comment by garrett** [comment]
@KKoukiou: Awesome! Thanks! Tag doesn't _need to be_ that long, but it's fine and I'm happy with it.

**Comment by garrett** [comment]
I've looked all over and haven't been able to find any more issues. :+1: Nice work!

File: pkg/machines/components/diskEdit.jsx [hunk 1]
```diff
@@ -208,6 +203,7 @@ class EditDiskModalBody extends React.Component {
                     <Modal.Title> {'Edit ' + getDiskPrettyName(vm.disks[disk.target]) + ' Attributes'} </Modal.Title>
                 </Modal.Header>
                 <Modal.Body>
+                    { showFooterWarning() }
```
Comment (marusak) [inline pkg/machines/components/diskEdit.jsx]: 'showFooterWarning` that does not show warning in footer but in body :) That is of course just detail ;)

File: pkg/docker/index.html [hunk 2]
```diff
@@ -557,7 +557,10 @@ <h4 class="modal-title" translate>Image Search</h4>
           </div>
         </div>
         <div class="modal-footer">
-          <input type="text" id="containers-search-tag" class="form-control" disabled="disabled" placeholder="Tag" />
+          <div class="ct-form containers-search-tag-form">
+            <label class='control-label' for="containers-search-tag" translate="yes">Tag</label>
+            <input type="text" id="containers-search-tag" class="form-control" disabled="disabled" placeholder="Tag" />
```
Comment (marusak) [inline pkg/docker/index.html]: placeholder should be translated as well.

Comment (KKoukiou) [reply pkg/docker/index.html]: Ups - this need to by mustache now then in order to pass the translated string. I'd rather drop the placeholder - it does not offer any information anyway.

Comment (marusak) [reply pkg/docker/index.html]: you can just do `translate="placeholder"`, or it does not work in this case?

Comment (KKoukiou) [reply pkg/docker/index.html]: I will leave this as is - as it's hard to get tests green and get the fixes for this and the function name in a followup

**Comment by martinpitt** [comment]
There is a conflict now, this needs rebasing.

File: pkg/docker/index.html (hunk 2 above)
Comment (martinpitt) [reply pkg/docker/index.html:562]: As this needs rebasing anyway, you may just as well do it now?

**Comment by martinpitt** [comment]
[fedora-31](https://logs.cockpit-project.org/logs/pull-13782-20200401-185504-5668a168-fedora-31/log.html#71) failed testAddDisk and all other Machines tests like this:

    internal error: child reported (status=125): Unable to get host boot time: No such process

@KKoukiou , @marusak: Did you see this before? I have a gut feeling that we did, but forgot how it was handled.

**Comment by martinpitt** [comment]
Now [testVmCreate](https://logs.cockpit-project.org/logs/pull-13782-20200401-212952-5668a168-fedora-31/log.html#202) fails again with the "usual" flake:

    Error validating install location: Could not find an installable distribution at 'https://archive.fedoraproject.org/pub/archive/fedora/linux/releases/28/Server/x86_64/os/': The URL could not be accessed, maybe you mistyped?

That seems to be fatal. I'll file a separate issue #13834 about this. I'm willing to trust that this is not due to the Cancel button.

**Comment by marusak** [comment]
>  Did you see this before? 

I see it now in https://github.com/cockpit-project/bots/pull/700 but really don't have any good idea what is the problem. Seems like the task container or so went bad.

**Pull request merged**